plugins/ptp_operator/metrics/manager.go

```diff
@@ -405,10 +405,11 @@ func (p *PTPEventManager) GenPTPEvent(ptpProfileName string, oStats *stats.Stats
 				oStats.AddValue(ptpOffset) // update off set when its in locked state and hold over only/ update off set when its in locked state and hold over only
 			} // else continue to stay in HOLDOVER
 		default: // not yet used states
-			log.Warnf("%s sync state %s, last ptp state is unknown: %s", eventResourceName, clockState, lastClockState)
-			if !isOffsetInRange(ptpOffset, threshold.MaxOffsetThreshold, threshold.MinOffsetThreshold) {
-				clockState = ptp.FREERUN
+			clockState = ptp.FREERUN
+			if isOffsetInRange(ptpOffset, threshold.MaxOffsetThreshold, threshold.MinOffsetThreshold) {
+				clockState = ptp.LOCKED
 			}
+			log.Warnf("%s sync state %s, last ptp state is unknown, setting to  %s", eventResourceName, clockState, lastClockState)
```

Comment:
this is wrong, should be
`log.Warnf("%s sync state LOCKED, last ptp state %s is unknown, setting to  %s", eventResourceName, lastClockState, clockState)`

Reply:
actually this log is not needed. you have all the info in the next log.

Reply:
resolved